packages/calcite-components/src/components/text-area/text-area.tsx

```diff
@@ -403,25 +403,37 @@ export class TextArea
   };
 
   renderCharacterLimit = (): VNode => {
-    return this.maxLength ? (
-      <span class={CSS.characterLimit}>
-        <span class={{ [CSS.characterOverLimit]: this.value?.length > this.maxLength }}>
-          {this.getLocalizedCharacterLength()}
+    if (this.maxLength) {
+      const { currentLength, maxLength } = this.getLocalizedCharactersLength();
+      return (
+        <span class={CSS.characterLimit}>
+          <span class={{ [CSS.characterOverLimit]: this.value?.length > this.maxLength }}>
+            {currentLength}
+          </span>
+          {"/"}
```

Comment:
Unrelated nitpick: We don't need to interpolate the string here, so can we use `/` directly or do you think it'd be confusing?

Reply:
i think its better to interpolate indicating it is part of the UI.